spec/views/admin/submissions/edit.html.erb_spec.rb

```diff
@@ -44,5 +44,12 @@
       expect(page).to have_content('Gob Bluth'.upcase)
       expect(page).to have_content('Add New')
     end
+
+    it 'lets you update the submission and not affect the assigned user' do
+      expect(@submission.assigned_to).to eq nil
+      find_button('Save').click
```

Comment:
**thought:** This is a hard thing for me to describe, so if I don't succeed, maybe we can talk over a slack call. My understanding of this test is that it's attempting to prove that when I save an unassigned submission, it saves but stays unassigned. I don't think the steps here are testing all of that, though. In particular, I think it's missing an assertion that the save was successful.

Without that assertion that the save was successful, this test wouldn't guarantee me that the `assigned_to` hadn't changed because we allow updates without choosing an `assigned_to` instead of the save flat out failing (and _nothing_ getting updated).

Reply:
To be more explicit about why I care about this: without that assertion that the save was successful, it's possible we'd have false green.

Reply:
We ended up chatting a bit about this and I addressed it with 2bd715d - does that look sufficient @pepopowitz?

Reply:
❤️ ❤️ ❤️